css-easing-2/Overview.bs

```diff
@@ -38,6 +40,14 @@ Repository: w3c/csswg-drafts
 }
 </pre>
 
+<style>
+  @media (prefers-color-scheme: dark) {
+    .easing-graph {
+      background: none;
```

Comment:
My intention was to remove the backgrounds so the graphs could be placed in notes/examples without being in their own visual 'box'.

As in, render like this:

![Screen Shot 2022-07-12 at 09 14 06](https://user-images.githubusercontent.com/93594/178443375-a68610f3-6309-4620-aec8-daf489fc2a12.png)

Not this:

![Screen Shot 2022-07-12 at 09 14 17](https://user-images.githubusercontent.com/93594/178443417-35360cf7-ea8c-4469-be56-a61a52067070.png)

But you would prefer the latter?

Reply:
@tabatkins _ping_. This is the last bit I'm unsure about.

Reply:
Your screenshot shows them against the lightmode note styling. Have you checked it against the darkmode note styling to confirm that it's actually readable? I suspect it's not.  (I'm fine with removing the background in lightmode, since they're authored to be readable against a light background.) I'm mostly just confused why you decided you needed to add an opaque background to some images, but explicitly *remove* an opaque background from others, when they look fairly similar in color usage to me.

Reply:
I use dark mode by default so I checked it there too (part of this PR is making those images work well in dark mode). The lack of consistency in the backgrounds is an error on my part, I'll fix them so they're all transparent.

Reply:
Here it is against darkmode styling. Seems readable to me.

![Screen Shot 2022-07-17 at 11 12 21](https://user-images.githubusercontent.com/93594/179393743-fdcee0e1-5b4f-4905-a8e1-3ac95ce12e2a.png)